### ISSUE TITLE
different size of molecules

### ISSUE THREAD

**Issue by nczub** [mhlee216/MGCVAE#4]
Hi,
I'm trying to use your code for a database with more atoms in molecules. Is it possible to change your code in this manner?

**Comment by mhlee216** [comment]
Try changing "--size" in argparse.
Good luck.

**Comment by nczub** [comment]
yes, I've already try this. When I changed this for example to 30 and I got this error:

  File "/data1/dane/MGCVAE_MULTI-OBJECTIVE_optimization/MGCVAE-main/code/mgcvae.py", line 168, in <module>
    feature_size(data[i], atom_labels, atom_number), 
  File "/data1/dane/MGCVAE_MULTI-OBJECTIVE_optimization/MGCVAE-main/Natalia/graph_converter.py", line 13, in feature_size
    feature = graph_features(mol, atom_labels, max_length)
  File "/data1/dane/MGCVAE_MULTI-OBJECTIVE_optimization/MGCVAE-main/Natalia/graph_converter.py", line 10, in graph_features
    return np.vstack((features, np.zeros((max_length - features.shape[0], features.shape[1]))))
ValueError: negative dimensions are not allowed
